modules/ROOT/pages/configure-monitoring-cloudhub.adoc

```diff
@@ -63,25 +78,25 @@ For supported versions of Mule, see xref:configure-monitoring-cloudhub.adoc[Conf
 
 You can use the *Settings* page in Anypoint Monitoring to enable or disable apps deployed to CloudHub and running on Mule.
 
-. Sign into Anypoint Platform and navigate to *Anypoint Monitoring*.
-. In the menu on the left, click *Settings*.
+. Sign in to Anypoint Platform and navigate to *Anypoint Monitoring*.
+. In the navigation menu, click *Settings*.
 +
 image::monitoring-settings-screen.png[]
 +
 . Select *CloudHub*.
-. In the *Environment* drop-down list, select the environment where the application is running. +
-A list of running applications is displayed for the selected environment. +
+. In the *Environment* list, select the environment where the application is running. +
+A list of running applications is shown for the selected environment. +
 You can search for a specific application using the *List of resources* search box.
```

Comment:
No; searching within that space is highlighted because there are users with oodles of apps who are unlikely to see their app in that list immediately when it opens. As such, it is noted that searching will suggest apps based on the characters entered. It's not like our site search where they enter an entire term, hit enter, and *then* the list is populated. They only need a piece of the app name for this to work.

This was a good question, though. I didn't write these original docs, so I like it when you bring up stuff like this since we're actively cleaning up Monitoring.